great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -449,19 +495,87 @@ def _get_round_decimals_using_heuristics(
         )
         if round_decimals is None:
             round_decimals = MAX_DECIMALS
-        elif not isinstance(round_decimals, int) or (round_decimals < 0):
-            raise ge_exceptions.ProfilerExecutionError(
-                message=f"""The directive "round_decimals" for {self.__class__.__name__} can be 0 or a
+        else:
+            if not isinstance(round_decimals, int) or (round_decimals < 0):
+                raise ge_exceptions.ProfilerExecutionError(
+                    message=f"""The directive "round_decimals" for {self.__class__.__name__} can be 0 or a
 positive integer, or must be omitted (or set to None).
 """
-            )
-        metric_value: Number
-        if all(
-            [
-                np.issubdtype(type(metric_value), np.integer)
-                for metric_value in metric_values
-            ]
-        ):
+                )
+
+        if np.issubdtype(metric_values.dtype, np.integer):
             round_decimals = 0
 
         return round_decimals
+
+    def _get_bootstrap_estimate(
+        self,
+        metric_values: np.ndarray,
+        domain: Domain,
+        *,
```

Comment:
Same point

Reply:
@cdkini Deleted (in next commit) -- thanks!